db/post_migrate/20191107190330_remove_suppress_from_latest_from_category.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class RemoveSuppressFromLatestFromCategory < ActiveRecord::Migration[6.0]
+  DROPPED_COLUMNS ||= {
+    categories: %i{suppress_from_latest}
+  }
+
+  def up
+    muted_category_ids = SiteSetting.default_categories_muted.split("|")
+    suppressed_category_ids = Category.where(suppress_from_latest: true).pluck(:id).map(&:to_s)
```

Comment:
we got to use direct SQL here with no access to the object model. Also if you find more than say 10 here do not add it to the default mute.

If you are adding to default mute you need to apply historically with all the logic of skipping if people explicitly watch/track.